src/app/Authenticated.tsx

```diff
@@ -1,56 +1,111 @@
-import { Skeleton } from '@mui/material';
+import { Collections } from '@mui/icons-material';
 import AppLayout from 'AppLayout';
+import CaptureCreate from 'components/capture/create';
+import CaptureEdit from 'components/capture/edit';
+import NewMaterialization from 'components/materialization/create';
+import Admin from 'pages/Admin';
+import Builds from 'pages/Builds';
+import Captures from 'pages/Captures';
 import PageNotFound from 'pages/error/PageNotFound';
 import Home from 'pages/Home';
-import { lazy, Suspense } from 'react';
+import Materializations from 'pages/Materializations';
 import { Route, Routes } from 'react-router';
 
-const Admin = lazy(() => import('../pages/Admin'));
-
-const Builds = lazy(() => import('../pages/Builds'));
-
-const Captures = lazy(() => import('../pages/Captures'));
-const CaptureCreate = lazy(() => import('components/capture/create/index'));
-const CaptureEdit = lazy(() => import('components/capture/edit/index'));
-
-const Materializations = lazy(() => import('../pages/Materializations'));
-const MaterializationCreate = lazy(
-    () => import('../components/materialization/create')
-);
-
-const Collections = lazy(() => import('../pages/Collections'));
+export const routeDetails = {
+    admin: {
+        title: 'routeTitle.admin',
+        path: '/admin',
+    },
+    builds: {
+        title: 'routeTitle.builds',
+        path: '/test/builds',
+    },
+    capture: {
+        root: '/capture',
+        create: {
+            title: 'routeTitle.captureCreate',
+            path: `create`,
+            fullPath: '/capture/create',
+        },
+        edit: {
+            title: 'routeTitle.captureEdit',
+            path: 'edit',
+            fullPath: '/capture/edit',
+        },
+    },
+    captures: {
+        title: 'routeTitle.captures',
+        path: '/captures',
+    },
+    collections: {
+        title: 'routeTitle.collections',
+        path: '/collections',
+    },
+    home: {
+        title: 'routeTitle.dashboard',
+        path: '/',
+    },
+    materialization: {
+        root: '/materialization',
+        create: {
+            title: 'routeTitle.materializationCreate',
+            path: 'create',
+        },
+    },
+    materializations: {
+        title: 'routeTitle.materializations',
+        path: '/materializations',
+    },
+    pageNotFound: {
+        title: 'routeTitle.error.pageNotFound',
+        path: '*',
+    },
+};
 
 const Authenticated = () => {
     return (
-        <Suspense fallback={<Skeleton animation="wave" />}>
```

Comment:
Removing suspense as it is _around_ the `AppLayout` so the entire page disappears and that is horrible UX. So this is good enough for now.